spec/serializers/v3/product_serializer_spec.rb

```diff
@@ -11,6 +11,26 @@
   end
   let(:base_url) { 'http://example.com' }
 
+  describe 'eula_url' do
+    it 'handles empty string' do
+      product = create(:product, :extension, base_products: [sles15], recommended: true, eula_url: '')
```

Comment:
any reason you are creating an extension & recommended product? i don't think that `#eula_url` cares about that.